docker-compose.yml

```diff
@@ -86,6 +88,28 @@ services:
       - DATABASE_PASSWORD=testkeycloakpassword
     networks:
       - safe-tier
+    volumes:
+      - program-artifacts:/usr/src/app/media/
+    depends_on:
+      - keycloak
+      - postgres
+  gateway_scheduler:
+    container_name: gateway_scheduler
+    image: icr.io/quantum-public/quantum-serverless-gateway:${VERSION:-0.1.0}
+    entrypoint: "./scripts/scheduler.sh"
+    environment:
+      - DEBUG=0
```

Comment:
same nit re: debug value... maybe we turn debug off for this one and on for the dev one?